File: go/B5_28113.go
// https://www.acmicpc.net/problem/28113
package main

import (
	"fmt"
)

func main() {
	var n, a, b int
	fmt.Scan(&n, &a, &b)
	ans := "Anything"
	if a > b {
		ans = "Subway"
	}
	if n > b || a < b {
		ans = "Bus"
	}
	fmt.Println(ans)
}

File: clang/B5_28113.c
//// https://www.acmicpc.net/problem/28113

#include <stdio.h>

int main(void)
{
	int n, a, b;
	scanf("%d %d %d", &n, &a, &b);
	if (n > b) puts("Bus");
	else if (a > b) puts("Subway");
	else if (a < b) puts("Bus");
	else puts("Anything");
	return 0;
}
